app/views/topics/show.html.erb

```diff
@@ -11,7 +11,7 @@
         itemref="breadcrumb-<%=(i+1)%>"
       <%-end%>>
       <a href="<%=c[:url]%>" itemprop="url" class='badge-wrapper bullet'>
-        <span class="badge-category-bg"></span>
+        <span class="badge-category-bg" style='background-color: #<%= c[:color] %>'></span>
```

Comment:
how is this going to work? where is the foreground? 

My call is not to even style this cause its just too annoying.